### PR TITLE
Update to lazy_static 0.2

### PR DESCRIPTION
**Pull request by mbrubeck**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,6 +10,8 @@ repository = "https://github.com/nagisa/rust_libloading/"
 documentation = "https://nagisa.github.io/rust_libloading/"
 
 [dependencies]
+lazy_static = "0.2"
+
+[target.'cfg(windows)'.dependencies]
```
Comment (nagisa) [inline Cargo.toml]: Woah, this is weird. Which versions of cargo support this?

Comment (nagisa) [inline Cargo.toml]: I guess we’ll see when the appveyor run completes.

Comment (mbrubeck) [inline Cargo.toml]: `target.cfg(...).dependencies` is supported starting in cargo 0.9.0 (Rust 1.8.0).

Comment (nagisa) [inline Cargo.toml]: Yeah, I feel like maintaining the support for, say, 1.6 is somewhat more desirable than avoiding a download and parse of winapi.

I think I’d be fine with taking something like this a few more versions of Rust later. Please fill an issue/suggest the change in a separate PR, lest I forget about it.

**Comment by nagisa** [comment]
I merged the first commit as https://github.com/nagisa/rust_libloading/commit/543f2086f05ecaa7420b47bb1252c73e38ba899d. Thanks!

**Comment by mbrubeck** [comment]
Submitted #10 for later consideration.  Thanks!

**Pull request closed without merging**